mesonbuild/coredata.py

```diff
@@ -332,6 +327,15 @@ def validate_option_value(self, option_name, override_value):
                 return opt.validate_value(override_value)
         raise MesonException('Tried to validate unknown option %s.' % option_name)
 
+    def external_args(self, lang):
+        return self.compiler_options[lang + '_args'].value
+
+    def external_link_args(self, lang):
+        return self.compiler_options[lang + '_link_args'].value
+
+    def external_preprocess_args(self, lang):
```

Comment:
Shouldn't this be `get_external_*()`? That's how class methods usually are.

Reply:
ok, done.